js/bibox.js

```diff
@@ -626,58 +628,82 @@ module.exports = class bibox extends Exchange {
 
     parseOHLCV (ohlcv, market = undefined) {
         //
-        //     {
-        //         "time":1591448220000,
-        //         "open":"0.02507029",
-        //         "high":"0.02507029",
-        //         "low":"0.02506349",
-        //         "close":"0.02506349",
-        //         "vol":"5.92000000"
-        //     }
+        //    [
+        //        '1656702000000',      // start time
+        //        '19449.4',            // opening price
+        //        '19451.7',            // maximum price
+        //        '19290.6',            // minimum price
+        //        '19401.5',            // closing price
+        //        '73.328833',          // transaction volume
+        //        '1419466.3805812',    // transaction value
+        //        '45740585',           // first transaction id
+        //        2899                  // The total number of transactions in the range
+        //    ]
         //
         return [
-            this.safeInteger (ohlcv, 'time'),
-            this.safeNumber (ohlcv, 'open'),
-            this.safeNumber (ohlcv, 'high'),
-            this.safeNumber (ohlcv, 'low'),
-            this.safeNumber (ohlcv, 'close'),
-            this.safeNumber (ohlcv, 'vol'),
+            this.safeInteger (ohlcv, 0),
+            this.safeNumber (ohlcv, 1),
+            this.safeNumber (ohlcv, 2),
+            this.safeNumber (ohlcv, 3),
+            this.safeNumber (ohlcv, 4),
+            this.safeNumber (ohlcv, 5),
         ];
     }
 
-    async fetchOHLCV (symbol, timeframe = '1m', since = undefined, limit = 1000, params = {}) {
+    async fetchOHLCV (symbol, timeframe = '1m', since = undefined, limit = undefined, params = {}) {
```

Comment:
@samgermain don't know why but I'm always getting this error
![image](https://user-images.githubusercontent.com/43336371/183628673-6e12b546-0901-4e43-af14-11e2a2fb4431.png)

Reply:
Can you try again? I am not getting any error. Also in the future, can you copy and paste the full verbose output (minus any sensitive information)instead of posting a screenshot of it?

```
% bibox fetchOHLCV ADA/USDT --verbose     
2022-08-09T23:47:11.261Z
Node.js: v18.4.0
CCXT v1.91.75
(node:3171) ExperimentalWarning: The Fetch API is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
bibox.fetchOHLCV (ADA/USDT)
fetch Request:
 bibox GET https://api.bibox.com/api/v4/marketdata/candles?symbol=ADA_USDT&time_frame=1m 
RequestHeaders:
 { 'content-type': 'application/json' } 
RequestBody:
 undefined 

handleRestResponse:
 bibox GET https://api.bibox.com/api/v4/marketdata/candles?symbol=ADA_USDT&time_frame=1m 200 OK 
ResponseHeaders:
 {
  'Alt-Svc': 'h3=":443"; ma=86400, h3-29=":443"; ma=86400',
  'Bibox-Time': '1660088834208',
  'Bibox-Usage': '1:3686:6314',
  'Cf-Cache-Status': 'DYNAMIC',
  'Cf-Ray': '...',
  Connection: 'keep-alive',
  'Content-Encoding': 'br',
  'Content-Type': 'text/plain;charset=UTF-8',
  Date: 'Tue, 09 Aug 2022 23:47:14 GMT',
  'Expect-Ct': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"',
  Server: 'cloudflare',
  'Strict-Transport-Security': 'max-age=31536000; includeSubDomains; preload',
  'Transfer-Encoding': 'chunked',
  Vary: 'Accept-Encoding',
  'X-Ca-Request-Id': '...',
  'X-Content-Type-Options': 'nosniff'
} 
ResponseBody:
 {"t":60000,"e":[["1660082880000","0.516671","0.516871","0.516571","0.516671","516.94","267.10329874","2544588",6],["1660082940000","0.516771","0.516771","0.516671","0.516671","418.01","216.00151071","2544594",3],["1660083000000","0.516671","0.517271","0.516671","0.517271","7983.79","4127.16250675","2544597",10],["1660083060000","0.517171","0.517271","0.517171","0.517271","378.39","195.71470569","2544607",3],["1660083120000","0.517371","0.517471","0.516971","0.516971","3278.25","1695.75485375","2544610",7],["1660083180000","0.516971","0.516971","0.516971","0.516971","351.24","181.58089404","2544617",3],["1660083240000","0.517071","0.517171","0.517071","0.517071","425.94","220.24740974","2544620",4],["1660083300000","0.517171","0.517171","0.516571","0.516571","564.98","292.00237458","2544624",5],["1660083360000","0.516671","0.517471","0.516571","0.517471","1976.63","1021.68422573","2544629",7],["1660083420000","0.517371","0.517471","0.517371","0.517371","748.58","387.33687118","2544636",8],["1660083480000","0.517471","0.517671","0.517471","0.517571","517.81","267.99957151","2544644",4],["1660083540000","0.517571","0.517671","0.517371","0.517371","2414.38","1249.59671738","2544648",5],["1660083600000","0.517371","0.517471","0.517371","0.517371","492.39","254.75272469","2544653",5],["1660083660000","0.517471","0.517571","0.517371","0.517571","508.5","263.1476915","2544658",5],["1660083720000","0.517471","0.517671","0.517471","0.517671","549.54","284.44313334","2544663",4],["1660083780000","0.517571","0.517971","0.517571","0.517971","1713.64","887.15348244","2544667",4],["1660083840000","0.517871","0.517871","0.517371","0.517671","2577.03","1333.61389513","2544671",8],["1660083900000","0.517471","0.517471","0.517271","0.517271","511.86","264.82091206","2544679",5],["1660083960000","0.517271","0.517571","0.517271","0.517571","499.09","258.27839439","2544684",5],["1660084020000","0.517471","0.517471","0.516871","0.516871","3077.93","1592.38528403","2544689",5],["1660084080000","0.516871","0.516871","0.516371","0.516371","2850.33","1472.57461943","2544694",10],["1660084140000","0.516471","0.516571","0.516471","0.516471","355.49","183.61679579","2544704",4],["1660084200000","0.516471","0.516471","0.516371","0.516371","441.53","228.00490463","2544708",4],["1660084260000","0.516471","0.516571","0.51627","0.51627","425.13","219.54856128","2544712",4],["1660084320000","0.51627","0.516471","0.51627","0.516471","424.78","219.33977008","2544716",4],["1660084380000","0.516371","0.516671","0.516371","0.516471","1787.03","922.96207113","2544720",6],["1660084440000","0.516571","0.516571","0.516471","0.516471","261.58","135.11087418","2544726",2],["1660084500000","0.516671","0.516671","0.516371","0.516371","1785.73","922.40785883","2544728",5],["1660084560000","0.516471","0.516471","0.516371","0.516371","445.06","229.84730426","2544733",4],["1660084620000","0.516371","0.516671","0.516371","0.516671","1843.52","952.16101792","2544737",8],["1660084680000","0.516671","0.516771","0.516671","0.516771","359.1","185.5604241","2544745",4],["1660084740000","0.516671","0.516771","0.51597","0.51597","557.77","287.99826194","2544749",5],["1660084800000","0.51587","0.51607","0.51557","0.51557","1864.98","962.2069826","2544754",5],["1660084860000","0.51557","0.51557","0.51467","0.51467","3271.07","1684.64028714","2544759",10],["1660084920000","0.51447","0.51457","0.513669","0.513769","5373.55","2762.37623548","2544769",7],["1660084980000","0.513269","0.513469","0.513269","0.513369","618.9","317.7100441","2544776",6],["1660085040000","0.513269","0.513269","0.511768","0.511968","3363.13","1724.06196092","2544782",7],["1660085100000","0.512368","0.512769","0.512268","0.512669","617.46","316.44446459","2544789",6],["1660085160000","0.512969","0.513069","0.512969","0.513069","478.96","245.71836224","2544795",5],["1660085220000","0.513269","0.513269","0.512669","0.512669","606.46","311.10030174","2544800",6],["1660085280000","0.512669","0.512669","0.512268","0.512551","4558.35","2335.62026747","2544806",9],["1660085340000","0.512769","0.512869","0.512569","0.512869","443.62","227.46163578","2544815",4],["1660085400000","0.513069","0.513369","0.513069","0.513369","594.84","305.27508696","2544819",7],["1660085460000","0.513469","0.513669","0.513469","0.513569","506.24","259.99383756","2544826",4],["1660085520000","0.513769","0.51447","0.513769","0.51447","3393.43","1744.35240561","2544830",11],["1660085580000","0.51457","0.51457","0.514169","0.514369","504.16","259.33374251","2544841",4],["1660085640000","0.514169","0.514269","0.514169","0.514269","523.78","269.33594082","2544845",5],["1660085700000","0.514169","0.514169","0.514169","0.514169","321.36","165.23334984","2544850",3],["1660085760000","0.514269","0.51457","0.514269","0.51457","2399.21","1233.94658583","2544853",5],["1660085820000","0.51467","0.51477","0.51457","0.51477","471.56","242.6967932","2544858",4],["1660085880000","0.51467","0.51477","0.51467","0.51477","467.37","240.5776929","2544862",5],["1660085940000","0.51467","0.51487","0.51467","0.51477","547.26","281.7033662","2544867",5],["1660086000000","0.51477","0.51477","0.51447","0.51477","427.45","219.9917895","2544872",3],["1660086060000","0.51477","0.51477","0.514169","0.514169","618.4","318.0956118","2544875",6],["1660086120000","0.514269","0.51467","0.514269","0.51467","401.71","206.66099242","2544881",4],["1660086180000","0.51467","0.51467","0.514269","0.514369","409.73","210.78586481","2544885",3],["1660086240000","0.51447","0.51467","0.514369","0.51467","615.06","316.46293252","2544888",7],["1660086300000","0.51497","0.51567","0.51497","0.51567","568.85","293.1394525","2544895",6],["1660086360000","0.51567","0.51567","0.51557","0.51567","600.89","309.8422283","2544901",6],["1660086420000","0.51577","0.51577","0.51527","0.51527","1781.76","918.6180862","2544907",4],["1660086480000","0.51547","0.51547","0.51477","0.51477","622.93","320.8897731","2544911",7],["1660086540000","0.51507","0.51527","0.51467","0.51527","1775.54","914.2348828","2544918",6],["1660086600000","0.51497","0.51497","0.51477","0.51487","440","226.543289","2544924",4],["1660086660000","0.51497","0.51557","0.51497","0.51547","1911.91","985.4241677","2544928",6],["1660086720000","0.51497","0.51527","0.51497","0.51507","3261.45","1680.1259555","2544934",7],["1660086780000","0.51527","0.51547","0.51517","0.51547","3137.56","1616.7183722","2544941",5],["1660086840000","0.51527","0.51537","0.51527","0.51537","355.34","183.1238128","2544946",4],["1660086900000","0.51527","0.51557","0.51527","0.51537","475.57","245.1092469","2544950",4],["1660086960000","0.51527","0.51527","0.51467","0.51467","559.3","287.996269","2544954",5],["1660087020000","0.51467","0.51497","0.51467","0.51497","526.12","270.8583024","2544959",6],["1660087080000","0.51487","0.51487","0.51447","0.51457","526.72","271.1025014","2544965",5],["1660087140000","0.514369","0.514369","0.513569","0.513769","562.51","289.08040919","2544970",7],["1660087200000","0.513869","0.514269","0.513869","0.514069","1872.3","962.7732027","2544977",6],["1660087260000","0.514269","0.514369","0.514069","0.514069","600.54","308.82492026","2544983",6],["1660087320000","0.514369","0.514369","0.514169","0.514169","404.48","208.00063312","2544989",4],["1660087380000","0.514069","0.514169","0.514069","0.514169","458.18","235.57417042","2544993",5],["1660087440000","0.514269","0.514269","0.513969","0.513969","484.16","248.88644704","2544998",5],["1660087500000","0.513569","0.514369","0.513569","0.514369","1994.39","1024.87127391","2545003",8],["1660087560000","0.51447","0.51447","0.514169","0.514169","350.01","180.00717632","2545011",3],["1660087620000","0.514169","0.514269","0.514169","0.514269","342.28","176.00843432","2545014",3],["1660087680000","0.514269","0.514269","0.514169","0.514269","403.77","207.62416713","2545017",4],["1660087740000","0.514169","0.51457","0.514169","0.51457","1832.2","942.29017793","2545021",5],["1660087800000","0.514369","0.51447","0.513969","0.514369","4691.54","2412.07703761","2545026",9],["1660087860000","0.51447","0.51477","0.51447","0.51477","1702.82","876.1213054","2545035",5],["1660087920000","0.51487","0.51497","0.51477","0.51497","453.65","233.5494915","2545040",5],["1660087980000","0.51507","0.51517","0.51507","0.51517","554.64","285.7094818","2545045",4],["1660088040000","0.51537","0.51537","0.51527","0.51527","377.77","194.6664829","2545049",3],["1660088100000","0.51487","0.51497","0.51487","0.51487","1784.86","919.1222252","2545052",5],["1660088160000","0.51487","0.51517","0.51487","0.51507","1762.63","907.8700561","2545057",5],["1660088220000","0.51527","0.51527","0.51467","0.51467","1949.65","1004.2516275","2545062",7],["1660088280000","0.51467","0.51477","0.51457","0.51457","404.66","208.2827322","2545069",4],["1660088340000","0.51457","0.51457","0.514369","0.51447","1899.42","977.313263","2545073",6],["1660088400000","0.51457","0.51467","0.51457","0.51467","699.58","360.0023106","2545079",8],["1660088460000","0.51467","0.51487","0.51457","0.51487","468.5","241.136653","2545087",5],["1660088520000","0.51487","0.51487","0.51447","0.51447","530.38","272.9385946","2545092",6],["1660088580000","0.51457","0.51457","0.514369","0.514369","259.18","133.33499107","2545098",2],["1660088640000","0.51447","0.51447","0.514269","0.514269","1920.82","988.12606137","2545100",6],["1660088700000","0.514169","0.514369","0.514069","0.514069","1701.58","875.05012902","2545106",4],["1660088760000","0.514169","0.514269","0.514169","0.514269","388.91","199.99398479","2545110",3],["1660088820000","0.514269","0.514269","0.514269","0.514269","88.89","45.71337141","2545113",1]]} 

2022-08-09T23:47:14.200Z iteration 0 passed in 422 ms

1660082880000 | 0.516671 | 0.516871 | 0.516571 | 0.516671 |  516.94
...
1660088820000 | 0.514269 | 0.514269 | 0.514269 | 0.514269 |   88.89
100 objects
2022-08-09T23:47:14.200Z iteration 1 passed in 422 ms
```

Reply:
@samgermain, for some reason works now 🤷‍♂️

Reply:
It was probably a problem with bibox's server's seeing as the error message was "Internal Server Error"